src/fparser/one/tests/test_do_block_r814.py

```diff
@@ -48,43 +52,43 @@
 @pytest.mark.parametrize('name', [None, 'loop_name'])
 @pytest.mark.parametrize('label', [None, '123'])
 @pytest.mark.parametrize('control_comma', [False, True])
-@pytest.mark.parametrize('initial_expression',
-                         ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
 @pytest.mark.parametrize('terminal_expression',
                          ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
-@pytest.mark.parametrize('incrament_expression',
-                         ['1', '10', 'x+y', 'size(array)', 'size(this%array)'])
 @pytest.mark.parametrize('end_name', [None, 'loop_name', 'wrong_name'])
 @pytest.mark.parametrize('end_label', [None, '123', '456'])
-def test_do(name, label, control_comma, initial_expression,
-            terminal_expression, incrament_expression, end_name, end_label):
+def test_do(name, label, control_comma, terminal_expression,
+            end_name, end_label):
     # pylint: disable=redefined-outer-name, too-many-arguments, too-many-locals
     '''
-    Checks that the "do" loop parser understands all forms of the syntax.
+    Checks that the "do" loop parser understands the "for-next" variant of
+    the syntax. This is defined in BS ISO/IEC 1539-1:2010 with R814-R822.
+
+    TODO: Only the terminal expression is tested. This is a short-cut and
```

Comment:
Please also turn this into just a normal comment otherwise we have to have a new Issue and we aren't actively developing fparser1.

Reply:
On second thoughts, don't worry.